tests/fuzz/go.mod

```diff
@@ -3,3 +3,6 @@ module github.com/fluxcd/kustomize-controller/tests/fuzz
 // with fuzz dependencies.
 
 go 1.17
+
+replace github.com/fluxcd/kustomize-controller/api => ../../api
+replace github.com/fluxcd/kustomize-controller => ../../
```

Comment:
This shouldn't be needed, this go.mod is managed by https://github.com/fluxcd/kustomize-controller/commit/f224109d26e829fb55c251c96ce77bad6bc09f8e

Reply:
We should likely apply that patch to the other controllers where we did this.

Reply:
Yes, as it is right now, we run the fuzzing with the `latest` version of all our dependencies, making fuzzing not accurate and error prone.

Reply:
Reverted it, but added a note about what _actually_ manages the go.mod in the file.